example.py

```diff
@@ -0,0 +1,19 @@
+def add(a, b):
+    """Функция суммы."""
+    return a + b
+
+
+def subtract(a, b):
+    """Вычитает b из a.
+
+    Эта функция демонстрирует многострочный docstring в Google стиле.
+
+    Args:
+        a: первое число
+        b: второе число
+
+    Returns
+        результат вычитания a - b
+
+    """
+    return a - b
```

Comment:
```suggestion
    return a - b
```

Это же чисто тестовая ерунда```

```